script/cibuild

```diff
@@ -19,7 +19,7 @@ if [ -d /usr/share/rbenv/shims ]; then
     export PATH=/usr/share/rbenv/shims:$PATH
 fi
 ruby -v
-rm Gemfile.lock
+rm -f Gemfile.lock
```

Comment:
This LGTM! Could also put the `rm Gemfile.lock` between `set +e ... set -e` block if missing file is the issue causing CI to stop but this seems safe, I like it! :shipit: